BoredApp/Singleton/ActivitySearchCriteria.swift

```diff
@@ -0,0 +1,51 @@
+//
+//  ActivityCriteria.swift
+//  BoredApp
+//
+//  Created by user on 21.09.2021.
+//
+
+import UIKit
+
+class ActivitySearchCriteria: NSObject {
+    
+    static let shared = ActivitySearchCriteria()
+    
+    private override init() {
+        self.types = [.social]
+        self.price = 0.1
+        self.participants = 2
+        super.init()
+    }
+    
+    private let service = Service()
```

Comment:
```suggestion
    private let service: ServiceProtocol = Service()
```
иначе смысл пропадает от интерфейса задекларированного